Makefile.dep

```diff
@@ -52,6 +52,11 @@ ifneq (,$(filter ng_ipv6_nc,$(USEMODULE)))
     USEMODULE += ng_ipv6_addr
 endif
 
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+    USEMODULE += ng_netapi
+    USEMODULE += ng_pktbuf
```

Comment:
How about adding 'ng_netreg' here as well? Although it's not directly needed for the shell commands, it is used by all other link-layer modules (drivers, MAC layers)